content/docs/user-guide/assets/asset-processor/settings.md

```diff
@@ -0,0 +1,46 @@
+---
+linkTitle: Settings
+title: Asset Processor Settings
+description: Commandline settings for AssetProcessor and AssetBuilder
```

Comment:
I don't see settings for AssetBuilder in this page. Do we mean AssetProcessorBatch?

Reply:
Fixed. Good catch